modules/db/src/main/java/org/mule/module/db/internal/domain/connection/OracleDbConnection.java

```diff
@@ -42,6 +42,7 @@
 
     private Method createArrayMethod;
     private boolean initialized;
+    private Map<String, Map<Integer, ResolvedDbType>> resolvedDbTypesCache= new HashMap<>();
```

Comment:
this cache should never be created here. It must always be received in the constructor

Reply:
Done